src/drivers/indexeddb.js

```diff
@@ -947,36 +951,57 @@ function dropInstance(options, callback) {
     if (!options.name) {
         promise = Promise.reject('Invalid arguments');
     } else {
-        var dbPromise =
-            options.name === currentConfig.name && self._dbInfo.db
-                ? Promise.resolve(self._dbInfo.db)
-                : _getOriginalConnection(options);
+        const isCurrentDb =
+            options.name === currentConfig.name && self._dbInfo.db;
+
+        const dbPromise = isCurrentDb
+            ? Promise.resolve(self._dbInfo.db)
+            : _getOriginalConnection(options).then(function(db) {
```

Comment:
Any reason we're using `function(db) {` here over `(db) => {`? I'm just used to seeing the more ES6-y type functions these days so I'd prefer we use them in the future if we can.

Reply:
TBH, I was just doing this for consistency in the code style :) Will be more than happy to change these.